magenta-lib/src/test/scala/magenta/input/DeploymentResolverTest.scala

```diff
@@ -0,0 +1,298 @@
+package magenta.input
+
+import org.scalatest.{FlatSpec, ShouldMatchers}
+import play.api.libs.json.{JsNumber, JsString, JsValue}
+
+class DeploymentResolverTest extends FlatSpec with ShouldMatchers {
+  "DeploymentResolver" should "parse a simple deployment with defaults" in {
+    val yaml = RiffRaffYaml(None, None, None,
+      Map("test" -> deploymentType("testType").withParameters("testParam" -> JsString("testValue")).withStacks("testStack")))
+    val deployments = DeploymentResolver.resolve(yaml)
+    deployments.size should be (1)
+    deployments.head.isRight should be(true)
+    val deployment = deployments.head.right.get
```

Comment:
scalatest can check multiple properties on an object at once.

deployments should have (
  'size (1),
  'type ("testType"),
  'stacks (List("testStack")),
  ... etc
)

It's much less verbose and (I think) easier to take in at a glance.

I think you'll still need to do other property tests that don't check a simple value (like `contain` / `isRight`) separately.